src/radar-chart/index.js

```diff
@@ -28,6 +28,7 @@ import XYPlot from 'plot/xy-plot';
 import {DISCRETE_COLOR_RANGE} from 'theme';
 import {MarginPropType} from 'utils/chart-utils';
 import PolygonSeries from 'plot/series/polygon-series';
+import CustomSVGSeries from 'plot/series/custom-svg-series';
```

Comment:
I think you can do this more clearly (and probably just more easily) by using a MarkSeries to capture the points. I think it would maybe be also helpful to have a voronoi option

Reply:
Awesome, I didn't know that was possible! Updated 👍